modules/data-mover-for-csi-snapshots.adoc

```diff
@@ -0,0 +1,197 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="data-mover-for-csi-snapshots_{context}"]
+= Data Mover for CSI snapshots
+
+
+The OADP 1.1 Data Mover enables customers to back up CSI volume snapshots to a remote object store and to restore stateful applications from the store in the event of a failure, accidental deletion or corruption of the cluster.
+The OADP 1.1 Data Mover solution uses the Restic option of VolSync.
+
+[NOTE]
+=====
+Data Mover supports backup and restore of CSI volume snapshots only.
+=====
+
+.Prerequisites
+
+* Have a stateful application running in a separate namespace.
+
+* The `StorageClass` and `VolumeShapshotClass` CRs support CSI.
+
+* Only one `volumeSnapshotClass` CR has the annotation `snapshot.storage.kubernetes.io/is-default-class: true`.
+
+* Only one `storageClass` CR has the annotation `storageclass.kubernetes.io/is-default-class: true`.
+
+* Include the label `{velero-domain}/csi-volumesnapshot-class: 'true'` in your `VolumeSnapshotClass` CR to indicate for Velero which to use.
+
+* Install the VolSync Operator using the Operator Lifecycle Manager (OLM). Administrators can link:https://docs.openshift.com/container-platform/4.10/operators/admin/olm-adding-operators-to-cluster.html[install Operators on clusters]. Non-administrators can link:https://docs.openshift.com/container-platform/4.10/operators/user/olm-installing-operators-in-namespace.html[install Operators in namespaces].
++
+[NOTE]
+====
+The VolSync Operator is required only for use with the Technology Preview Data Mover. The Operator is not required for using OADP GA features.
+====
+
+* Install the link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md[OADP Operator] using OLM.
+
+* Install the link:https://volsync.readthedocs.io/en/stable/installation/index.html[VolSync operator] using OLM.
+
+.Procedure
+
+. Configure a link:https://volsync.readthedocs.io/en/stable/usage/restic/index.html#id2[Restic secret].
```

Comment:
don't think we need to refer this link, cause as part of the logic the specific creds data is created dynamically behind the scene, so the user just need to set on the secret `RESTIC_PASSWORD`, and the rest of the configuration is taken from the BSL configuration.
@savitharaghunathan, @shubham-pampattiwar  please correct me if wrong

Reply:
+1 to what Maya said.

Reply:
@mperetzred  So the entire first step is unnecessary?

Reply:
@sbeskin-redhat The step of creating a restic secret is still necessary, but because we only need the password, that link is unnecessary now because it include additional fields. The example secret below this link should suffice.

Reply:
+1, you could just say, configure a datamover secret and supply a `RESTIC_PASSWORD` via this secret in the operator installation namespace. Also, add a note that by default the operator looks for `dm-credential` named secret, if you are using a different name then you need to specify the name via DPA CR using `dpa.spec.features.dataMover.credentialName`

Reply:
@shubham-pampattiwar 
"..configure a datamover secret..." - or Restic secret?